modules/api/pkg/handler/v1/admin/settings.go

```diff
@@ -140,7 +142,7 @@ func convertAPISettingsToSettingsSpec(settings *apiv2.GlobalSettings) (kubermati
 	}
 
 	if settings.DefaultProjectResourceQuota != nil {
-		crdQuota, err := apiv2.ConvertToCRDQuota(*settings.DefaultProjectResourceQuota)
+		crdQuota, err := apiv2.ConvertToCRDQuota(settings.DefaultProjectResourceQuota.Quota)
```

Comment:
Do we have any validation to be sure that Quota is not `nil` in this case?

Reply:
It should fail at the marshalling phase in case is no quota, right?

Reply:
It's a struct so it should work fine, will be an empty object. `Quota` is not a pointer.